src/reference/environments/example_envs.rst

```diff
@@ -0,0 +1,49 @@
+Example Environments
+====================
+
+A set of possible installations of Cylc:
+
+.. csv-table::
+   :header:  environment, cylc, cylc uiserver, rose
+
+   `Cylc 8 basic`_, ✔️, ,
+   `Cylc 8 with UI Server`_, ✔️,✔️,
+   `Cylc 8 with Rose`_, ✔️,✔️,✔️
+
+Example installation commands:
+
+.. code-block:: bash
+
+   conda env create -f path/to/env.yml --name "name of env"
+
+   # ..or if you haven't specified a python version in the enviroment file:
+   conda env create -f path/to/env.yml --name "name of env" python==<version>
+
+
+Cylc 8 basic
+------------
+
+In this case we are collecting the core workflow engine.
+
+Also demonstrates using different versions of Python.
+
+.. literalinclude:: basic-3.7.yml
+   :language: YAML
+
+Cylc 8 with UI Server
+---------------------
+
+Create an enviroment containing the workflow engine and the GUI components
+from releases available on conda-forge.
```

Comment:
typo "enviroment".

```suggestion
An environment containing the workflow engine and the GUI components.
```

(all recipes use releases on conda-forge so it might be confusing stating that here).